appveyor.yml

```diff
@@ -148,7 +148,8 @@ before_test:
      Push-AppveyorArtifact $installerLogFilePath
      $crashDump = "$outputDir\nvda_crash.dmp"
      if (Test-Path -Path $crashDump){
-      Push-AppveyorArtifact $crashDump
+      Rename-Item -Path $crashDump -NewName "nvda_install_crash.dmp"
+      Push-AppveyorArtifact "$outputDir\nvda_install_crash.dmp"

```

Comment:
[`Push-AppveyorArtifact` has a `-FileName` argument to set the name of the artifact](https://www.appveyor.com/docs/build-worker-api/#push-artifact). Essentially renaming it.